### DISCUSSION TITLE
V11.299 stable release - keep getting: License signature server was unable to be reached

### DISCUSSION THREAD

**Discussion by Bekradan** [felddy/foundryvtt-docker#695] [General]
Hi guys,

I've tried this two or three times now and I keep getting the same error.  Any ideas?

I have 11.295 installed as a container and that worked like a charm and continues to do so.  Likewise for the last v10 and v9 versions.

Sorry I'm not able to provide greater technical detail as installing one of these via my QNAP Nas is about the limit of my abilities. 

Thanks in advance for any help you can offer.

**Comment by i-am-rayo** [comment]
Hello @Bekradan ,

Were you able to resolve this issue?
Now I find myself stuck trying to resolve it as well.

Hello @felddy,

Would you please advise if there is anything that can be done from our end?

Thank you in advance for your assistance in this matter.

**Reply by felddy** [reply]
The best I can do without more information is take a guess...

This sounds like a DNS issue to me.   Possibly related to IPv6.  

There has been some chatter on the discord about the `AAAA` for foundryvtt.com disappearing recently which could cause this behavior for some users. 

I can confirm that currently foundryvtt.com does not have an IPv6 address:

```console
➤ dig foundryvtt.com A foundryvtt.com AAAA +short
18.237.161.213
```

See: 
- https://discord.com/channels/170995199584108546/978002307251863552/1127126925274533979

**Reply by i-am-rayo** [reply]
@felddy ,

Thank you for your prompt response and feedback. 
After a few hours figure it out!

Since I don't access my VPS, a few things were not working properly.
Managed to get it back up and running.

**Reply by felddy** [reply]
@Daitsuke I'm glad you figured it out.  Anything about your solution you'd like to share for posterity?  😄

**Reply by i-am-rayo** [reply]
Yes, for sure.

One of the major issues was that I forgot that I changed the domain name, so the proxy was still with the old one; therefore, it could not connect to the check the EULA.
